Lib/test/test_symtable.py

```diff
@@ -54,33 +181,70 @@ class SymtableTest(unittest.TestCase):
     top = symtable.symtable(TEST_CODE, "?", "exec")
     # These correspond to scopes in TEST_CODE
     Mine = find_block(top, "Mine")
+
     a_method = find_block(Mine, "a_method")
     spam = find_block(top, "spam")
     internal = find_block(spam, "internal")
     other_internal = find_block(spam, "other_internal")
     foo = find_block(top, "foo")
-
+    Alias = find_block(top, "Alias")
+    GenericAlias = find_block(top, "GenericAlias")
+    # TODO(RUSTPYTHON)
+    # GenericAlias_inner = find_block(GenericAlias, "GenericAlias") 
+    generic_spam = find_block(top, "generic_spam")
+    # TODO(RUSTPYTHON)
+    # generic_spam_inner = find_block(generic_spam, "generic_spam")
+    GenericMine = find_block(top, "GenericMine")
+    # TODO(RUSTPYTHON)
+    # GenericMine_inner = find_block(GenericMine, "GenericMine")
+    # TODO(RUSTPYTHON)
+    # T = find_block(GenericMine, "T")
+    # TODO(RUSTPYTHON)
+    # U = find_block(GenericMine, "U")
+
+
+    # TODO(RUSTPYTHON)
```

Comment:
```suggestion
    # TODO: RUSTPYTHON
```
The notation itself looks good.
But we use search tools for this comment. Keeping consistency will be helpful